modules/compute-vm/variables.tf

```diff
@@ -173,12 +173,15 @@ variable "gpu" {
           "nvidia-tesla-p4",
           "nvidia-tesla-t4",
           "nvidia-l4",
-          "nvidia-a2"
+          "nvidia-a2",
+          "nvidia-tesla-t4-vws",
```

Comment:
Thanks for this, I just checked what gcloud says, and I have following list:
```
nvidia-a100-80gb
nvidia-h100-80gb
nvidia-h100-mega-80gb
nvidia-h200-141gb
nvidia-l4
nvidia-l4-vws
nvidia-tesla-a100
nvidia-tesla-p100
nvidia-tesla-p100-vws
nvidia-tesla-p4
nvidia-tesla-p4-vws
nvidia-tesla-t4
nvidia-tesla-t4-vws
nvidia-tesla-v100
```

`nvidia-a2` looks like an error, and  `nvidia-tesla-k80` was depreated [last year](https://cloud.google.com/compute/docs/eol/k80-eol)

Reply:
Yeah, updated the list now.